python/additions.py

```diff
@@ -1,7 +1,7 @@
 # Solve the quadratic equation ax**2 + bx + c = 0
 
 # import complex math module
-import cmath
+imprt cmath
```

Comment:
```suggestion
import cmath
```

<p>Fixed a typo...<br></p>

Reply:
<p><span data-name="grinning_face_with_closed_eyes" class="text-link px-1" data-type="emoji">😄</span><br></p>

Reply:
<p>dafdadf<br></p>

Reply:
<p><br></p>